src/library/scala/collection/SortedMap.scala

```diff
@@ -174,7 +174,7 @@ trait SortedMapOps[K, +V, +CC[X, Y] <: Map[X, Y] with SortedMapOps[X, Y, CC, _],
   override def concat[V2 >: V](xs: Iterable[(K, V2)]): CC[K, V2] = sortedMapFactory.from(new View.Concat(toIterable, xs))
 
   /** Alias for `concat` */
-  @`inline` override final def ++ [V2 >: V](xs: Iterable[(K, V2)]): CC[K, V2] = concat(xs)
+  @`inline` override def ++ [V2 >: V](xs: Iterable[(K, V2)]): CC[K, V2] = concat(xs)
```

Comment:
`@inline` without `final` will usually not be inlined, which leads to warnings when compiling user code.

Reply:
perhaps we could just literally inline the same implementation as concat if it's a concern? Sorry if that is a dumb suggestion :-)